ipfs-cluster-ctl/main.go

```diff
@@ -271,14 +266,14 @@ although unpinning operations in the cluster may take longer or fail.
 				{
 					Name:  "ls",
 					Usage: "List tracked CIDs",
-					UsageText: `
+					Description: `
 This command will list the CIDs which are tracked by IPFS Cluster and to
 which peers they are currently allocated. This list does not include
 any monitoring information about the IPFS status of the CIDs, it
 merely represents the list of pins which are part of the shared state of
 the cluster. For IPFS-status information about the pins, use "status".
 `,
-					ArgsUsage: "[cid]",
+					ArgsUsage: "[CID]",
```

Comment:
Should this be blank (`ArgsUsage: ""`), or does `pin ls` take in a list for filtering results or something?